src/components/app/content/context.js

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+const Context = React.createContext()
```

Comment:
Ideally you should move cards data nd methods to process them into context file.